package.json

```diff
@@ -20,6 +20,7 @@
     "test:custom-lint-rule": "eslint tools/eslint/rules/valid-constructors.test.ts",
     "test:dev": "jest --detectOpenHandles --runInBand --watch",
     "prepare": "tsc",
+    "prepack": "cp -r src/bin/commands/template lib/bin/commands/template",
```

Comment:
I wonder if adding `src/bin/commands/template` to [`files`](https://docs.npmjs.com/cli/v8/configuring-npm/package-json#files) is a more intention revealing version of this? Especially as JSON files cannot be commented 😞 .

Reply:
Unfortunately that includes the files under `src/bin/commands/template`, but we need them under `lib/bin/commands/template`